Dockerfile

```diff
@@ -6,9 +6,23 @@ USER root
 
 RUN apt-get update && apt-get install -y \
     # GCC required to resolve error during JupyterLab installation: psutil could not be installed from sources because gcc is not installed.
-    gcc \
+    gcc curl \
     && rm -rf /var/lib/apt/lists/*
 
+# Install jars to support delta lake spark operations
+ENV HADOOP_AWS_VER=3.3.4
+RUN curl -O https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/${HADOOP_AWS_VER}/hadoop-aws-${HADOOP_AWS_VER}.jar \
+    && mv hadoop-aws-${HADOOP_AWS_VER}.jar /opt/bitnami/spark/jars
+
+# NOTE: ensure Delta Spark jars matche python pip delta-spark version specified in the Pipfile
+ENV DELTA_SPARK_VER=3.2.0
+ENV SCALA_VER=2.12
+RUN curl -O https://repo1.maven.org/maven2/io/delta/delta-spark_${SCALA_VER}/${DELTA_SPARK_VER}/delta-spark_${SCALA_VER}-${DELTA_SPARK_VER}.jar \
+    && mv delta-spark_${SCALA_VER}-${DELTA_SPARK_VER}.jar /opt/bitnami/spark/jars
+
+Run curl -O https://repo1.maven.org/maven2/io/delta/delta-storage/${DELTA_SPARK_VER}/delta-storage-${DELTA_SPARK_VER}.jar \
+    && mv delta-storage-${DELTA_SPARK_VER}.jar /opt/bitnami/spark/jars
```

Comment:
None of these have dependencies? 

Even if not, I'd probably do this with Gradle. Setting up a dependencies only build file is pretty easy

Reply:
I honestly have barely no knowledge how Gradle works. Maybe I can learn it at a later time unless you think it's necessary to get it done now.

Reply:
It's not hard, but a TODO is fine

Reply:
:+1: